src/test/java/reposense/git/GitUtilTest.java

```diff
@@ -0,0 +1,46 @@
+package reposense.git;
+
+import static reposense.git.GitUtil.convertToGitExcludeGlobArgs;
+import static reposense.util.StringsUtil.addQuote;
+
+import java.io.File;
+import java.util.Collections;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import reposense.template.GitTestTemplate;
+
+public class GitUtilTest extends GitTestTemplate {
+
+    @Test
+    public void gitUtil_convertToGitExcludeGlobArgs_success() {
+        File repoRoot = new File(config.getRepoRoot());
+        final String cmdFormat = " " + addQuote(":(exclude)%s");
+        final String emptyResult = "";
+
+        String result = convertToGitExcludeGlobArgs(repoRoot, Collections.EMPTY_LIST);
+        Assert.assertEquals(result, emptyResult);
+
+        result = convertToGitExcludeGlobArgs(repoRoot, Collections.singletonList("**.js"));
+        Assert.assertEquals(result, String.format(cmdFormat, "**.js"));
```

Comment:
By convention, `expected` is the 1st argument, and `actual` is the 2nd argument when we use `assertEquals`.